rtgui/colortoning.cc

```diff
@@ -11,6 +11,8 @@ using namespace rtengine;
 using namespace rtengine::procparams;
 
 
+static constexpr int ID_LABREGION_HUE = 5;
```

Comment:
I did some research when I first encountered a  `static constexpr` in your code, but didn't come to a final conclusion. Is the `static` really needed? If I remember correctly, Bjarne Stroustrup voted for "no" while some others said "yes". What's your source for this?